repo/packages/M/marathon-lb/19/marathon.json.mustache

```diff
@@ -9,7 +9,15 @@
     "docker": {
       "image": "{{resource.assets.container.docker.marathon-lb-docker}}",
       "network": "HOST",
-      "privileged": true
+      "privileged": true,
+      "parameters": [
```

Comment:
Ok - @nlsun - (with a little help) I've updated the config.json to include this.

I was confused and looking at the marathon config.json, NOT the marathon-lb config.json!

Hope this helps guide in this change!